src/components/Tool/FixEmailAddresses/FixEmailAddresses.tsx

```diff
@@ -256,10 +334,12 @@ export const FixEmailAddresses: React.FC<FixEmailAddressesProps> = ({
                         handleSourceChange(event)
                       }
                     >
-                      <option value="MPDX">MPDX</option>
+                      <option value={appName}>{appName}</option>
                       <option value="DataServer">DataServer</option>
```

Comment:
Do we want DataServer to be translated? Or because it's a name, it should stay DataServer.

Reply:
This actually shouldn't be here at all. In the Angular version it is looping through an array with the existing sources plus `MPDX`:

```
union(
  flatMap((person) => map('source', person.email_addresses), data),
  ['MPDX'],
).sort();
```

Reply:
This will need to change for mailing addresses too.

Reply:
To answer the question of translation, they are not translated in the Angular app, so I think we can stick to the same here.